### DISCUSSION TITLE
Getting a ScreenLogic error on npm install

### DISCUSSION THREAD

**Discussion by subcritical71** [tagyoureit/nodejs-poolController#701] [App and O/S Config Questions]
Borked my install so I decided to reimage. Noticed that the latest pull gets me 8.0.0. 

When I use the command npm i, I get the following error;

npm ERR! code ETARGET
npm ERR! notarget No matching version found for node-screenlogic@^2.0.0.
npm ERR! notarget In most cases you or one of your dependencies are requesting
npm ERR! notarget a package version that doesn't exist.

**Comment by tagyoureit** [comment]
~~Ya, sorry -- waiting for the maintainer of the other repo to actually publish the 2.0 module.  Should be done tomorrow.  You can pull the 7.7 version for now.  https://www.npmjs.com/package/nodejs-poolcontroller/v/7.7.0.~~

This is published now.  You can pull again.

**Comment by subcritical71** [comment]
Great thanks. I'll be getting back to this this morning.

**Comment by subcritical71** [comment]
It's working now. Thanks. 

Another question about the install. In the past I would limit the nodejs to v14 as it seemed getting the latest and 'greatest' broke your controller. Now the minimum is v16 and that is what I installed. Has it been tested with the latest, v18 or v19?

**Reply by tagyoureit** [reply]
It will work with 16/18. Haven't tested with 19.

**Reply by rstrouse** [reply]
As an aside never install the odd numbered nodejs releases in a production environment.  These are considered pre-production.
